src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Is there anything in the system to detect discrepancy between the type of value added in clrconfigvalues.h and jitconfigvalues.h?